bin/build_utils

```diff
@@ -25,3 +25,25 @@ function gen_versions() {
     echo $version
   done
 }
+function git_tag_suffix() {
```

Comment:
please add an empty line before this function

Reply:
done